crates/rolldown/tests/esbuild/loader/loader_file_one_source_two_different_output_paths_css/artifacts.snap

```diff
@@ -4,18 +4,24 @@ snapshot_kind: text
 ---
 # Assets
 
-## common-YiR-P8G6.css
+## assets/common-h8VKzHVD.png
+
+## common-3yRNVswI.css
 
 ```css
 div {
-	background: url(common.png);
+	background: url(assets/common-h8VKzHVD.png);
 }
 
 ```
 ## common.js
 
 ```js
 
+//#region shared/common.png
```

Comment:
This redundant file is not related to this pr, since we will create `ecma_view` related to asset whatever before.